### PR TITLE
Document the use of trace flags when examining performance

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: doc/user/reporting-performance-problems.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ that is unique to us - it applies to many sophisticated virtual machines - but
 most Ruby implementations are not yet doing optimisations powerful enough to
 show them so it may be a new problem to some people in the Ruby community.
 
+## How to check for basic performance problems
+
+If you are examining the performance of TruffleRuby, we would recommend that you
+always run with the `-J-Dgraal.TraceTruffleCompilation=true` flag. If you see
+compilation failures or repeated compilation or the same methods, this is an
```
Comment (nirvdrum) [inline doc/user/reporting-performance-problems.md]: "...repeated compilation or the same methods..." -> "...repeated compilation of the same methods..."

File: doc/user/reporting-performance-problems.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ that is unique to us - it applies to many sophisticated virtual machines - but
...
+indicator that something is not working as intended and you may need to examine
+way, or ask us to help you do so. If you don't run with this flag Truffle will
```
Comment (nirvdrum) [inline doc/user/reporting-performance-problems.md]: "may need to examine way" -> "may need to examine why" ?

File: doc/user/reporting-performance-problems.md (hunk 1 above)
Comment (chrisseaton) [reply doc/user/reporting-performance-problems.md]: Fixed.

File: doc/user/reporting-performance-problems.md (hunk 2 above)
Comment (chrisseaton) [reply doc/user/reporting-performance-problems.md]: Fixed.

**Comment by chrisseaton** [comment]
Fixed.

**Pull request merged**